plugins/woocommerce-docs/package.json

```diff
@@ -27,6 +29,7 @@
 		"@types/react-dom": "^17.0.2",
 		"@woocommerce/dependency-extraction-webpack-plugin": "workspace:*",
 		"@woocommerce/eslint-plugin": "workspace:*",
+		"@wordpress/env": "^8.0.0",
```

Comment:
WooCommerce has `"@wordpress/env": "^8.2.0",` which I know doesn't make a difference but it may help to keep this the same when someone searches the codebase when updating in the future.

Reply:
Yep this should be updated, this changed with a recent PR. thanks for pointing this out.